app/models/ems_refresh/refreshers/container_refresher_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module EmsRefresh
+  module Refreshers
+    module ContainerRefresherMixin
+      def kubernetes_fetch_inventory(client)
+        fetch_entities(client, KUBERNETES_ENTITIES)
+      end
+
+      def openshift_fetch_inventory(client)
+        fetch_entities(client, OPENSHIFT_ENTITIES)
+      end
```

Comment:
Yep!  In fact, I think that's what @matthewd's [saying above](https://github.com/ManageIQ/manageiq/pull/5477/files#r47231952).  That would be better.

I was reacting more to the idea of grouping similar looking methods together instead of grouping common reusable methods together that are used by more than one component.